src/main/scala/edu/ie3/simona/model/participant/PvModel.scala

```diff
@@ -151,15 +138,14 @@ final case class PvModel private (
       calcReflectedRadiationOnSlopedSurface(eBeamH, eDifH, gammaE, albedo)
 
     // === Total Radiation ===//
-    val eTotal = eDifS.add(eBeamS.add(eRefS))
+    val eTotal = eDifS + eBeamS + eRefS
 
-    val irradiationSTC =
-      yieldSTC.multiply(duration).asType(classOf[Irradiation])
+    val irraditionSTC = yieldSTC * duration
     calcOutput(
       eTotal,
       data.dateTime,
-      irradiationSTC
-    )
+      irraditionSTC
+    ) * scalingFactor
```

Comment:
Good point, I also had to check this again. So far scaling is not done on the level of SystemParticipant but in the models. #81 tends to change this. Then it will be removed here and scaling is done for all participants the same way. But so far this scales the PVModel output by the scalingFactor